src/base/static/components/input-form/input-form-category-button.scss

```diff
@@ -0,0 +1,115 @@
+.input-form-category-button {
```

Comment:
again, let's try to flatten out the nesting here. I believe this can be implemented without any nesting at all.